examples/src/main/scala/org/apache/spark/examples/snappydata/CreatePartitionedRowTable.scala

```diff
@@ -0,0 +1,223 @@
+    // in SnappyStore when the data migrates. Here we specify
+    // the table to have 11 buckets
+    val props1 = Map("PARTITION_BY" -> "PS_PARTKEY", "BUCKETS" -> "11")
+    snc.createTable("PARTSUPP", "row", schema, props1)
+
+    pw.println("Inserting data in PARTSUPP table")
+    val data = Seq(Seq(100, 1, 5000, BigDecimal(100)),
+      Seq(200, 2, 50, BigDecimal(10)),
+      Seq(300, 3, 1000, BigDecimal(20)),
+      Seq(400, 4, 200, BigDecimal(30))
+    )
+    val rdd = snc.sparkContext.parallelize(data,
+      data.length).map(s => new Data(s(0).asInstanceOf[Int],
+      s(1).asInstanceOf[Int],
+      s(2).asInstanceOf[Int],
+      s(3).asInstanceOf[BigDecimal]))
+
+    val dataDF = snc.createDataFrame(rdd)
+    dataDF.write.insertInto("PARTSUPP")
+
+    pw.println("Printing the contents of the PARTSUPP table")
+    var tableData = snc.sql("SELECT * FROM PARTSUPP").collect()
+    tableData.foreach(pw.println)
+
+    pw.println()
+    pw.println("Update the available quantity for PARTKEY 100")
+    snc.update("PARTSUPP", "PS_PARTKEY = 100", Row(50000), "PS_AVAILQTY")
+
+    pw.println("Printing the contents of the PARTSUPP table after update")
+    tableData = snc.sql("SELECT * FROM PARTSUPP").collect()
+    tableData.foreach(pw.println)
+
+    pw.println()
+    pw.println("Delete the records for PARTKEY 400")
+    snc.sql("DELETE FROM PARTSUPP WHERE PS_PARTKEY = 400")
+    snc.delete("PARTSUPP", "PS_PARTKEY = 400")
+
+    pw.println("Printing the contents of the PARTSUPP table after delete")
+    tableData = snc.sql("SELECT * FROM PARTSUPP").collect()
+    tableData.foreach(pw.println)
+
+    pw.println("****Done****")
+  }
+
+  /**
+   * Creates a partitioned row table and performs operations on it using SQL queries
+   * thru SnappyContext
+   *
+   * Other way to execute a SQL statement is thru JDBC or ODBC driver. Refer to
+   * JDBCExample.scala for more details
+   */
+  def createPartitionedRowTableUsingSQL(snc: SnappyContext, pw: PrintWriter): Unit = {
+    pw.println()
+
+    pw.println("****Create a row table using SQL****")
+    // create a partitioned row table using SQL
+    pw.println()
+    pw.println("Creating a partitioned row table(PARTSUPP) using SQL")
+
+    snc.sql("DROP TABLE IF EXISTS PARTSUPP")
+
+    // Create the table using SQL command
+    // "PARTITION_BY" attribute specifies partitioning key for PARTSUPP table(PS_PARTKEY),
+    // "BUCKETS" attribute specifies the smallest unit that
+    // can be moved around in SnappyStore when the data migrates. Here we specify
+    // the table to have 11 buckets
+    // For complete list of table attributes refer the documentation
+    snc.sql("CREATE TABLE PARTSUPP ( " +
+        "PS_PARTKEY     INTEGER NOT NULL PRIMARY KEY," +
+        "PS_SUPPKEY     INTEGER NOT NULL," +
+        "PS_AVAILQTY    INTEGER NOT NULL," +
+        "PS_SUPPLYCOST  DECIMAL(15,2)  NOT NULL)" +
+        "USING ROW OPTIONS (PARTITION_BY 'PS_PARTKEY', BUCKETS '11' )")
```

Comment:
Remove buckets. We don't want users to be unnecessarily playing around with buckets. Keep in just one example, if required, in the entire set of examples/quickstart.